api/routes/Event.js

```diff
@@ -27,10 +27,9 @@ router.post('/createEvent', (req, res) => {
   } else if (!checkIfTokenValid(req)) {
     return res.sendStatus(UNAUTHORIZED)
   }
-  logger.log(req.body.id)
   const newEvent = new Event({
     title: req.body.title,
```

Comment:
fun fact, by doing

```js
const obj = {
   title
}
```

this automatically implies
```js
const obj = {
   title: title
}
```

change this `NewEvent` line by doing
```js
const {
  title, eventLocation, eventDate,
  /* all other fields here */
} = req.body
```
then do

```js
 const newEvent = new Event({
  title, eventLocation, eventDatem
   /* all other fields here */
})
```